packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-node-ciemss.vue

```diff
@@ -273,11 +274,24 @@ watch(
 			`;
 			const summaryResponse = await createLLMSummary(prompt);
 
-			const portLabel = props.node.inputs[0].label;
+			const parentModel = await getModelByModelConfigurationId(modelConfigId.value as string);
+			const calibratedModelConfig: ModelConfiguration = {
+				name: 'Calibrated configuration',
+				description: 'Calibrated configuration',
```

Comment:
I think those should use the same name as the original modelConfiguration but with a prepend like `Calibrated`.

Reply:
sure